src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/sink/SinkProviderTLSTest.java

```diff
@@ -129,15 +128,14 @@ private SSLContext createSSLContext() throws Exception {
 
 	private void assertStringContainsAllKeysAndValues(String in) {
 	    System.out.println(in);
-		Assert.assertTrue(in, in.contains(AuditMessage.FORMAT_VERSION));
-		Assert.assertTrue(in, in.contains(AuditMessage.CATEGORY));
-		Assert.assertTrue(in, in.contains(AuditMessage.FORMAT_VERSION));
-		Assert.assertTrue(in, in.contains(AuditMessage.REMOTE_ADDRESS));
-		Assert.assertTrue(in, in.contains(AuditMessage.ORIGIN));
-		Assert.assertTrue(in, in.contains(AuditMessage.REQUEST_LAYER));
-		Assert.assertTrue(in, in.contains(AuditMessage.TRANSPORT_REQUEST_TYPE));
-		Assert.assertTrue(in, in.contains(AuditMessage.UTC_TIMESTAMP));
-		Assert.assertTrue(in, in.contains(Category.FAILED_LOGIN.name()));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.FORMAT_VERSION));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.CATEGORY));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.REMOTE_ADDRESS));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.ORIGIN));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.REQUEST_LAYER));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.TRANSPORT_REQUEST_TYPE));
+		Assert.assertTrue(in, in.contains(AuditMessageBuilder.UTC_TIMESTAMP));
+		Assert.assertTrue(in, in.contains(AuditCategory.FAILED_LOGIN.name()));
```

Comment:
why .name() for this one?

Reply:
This is a category type (part of enum). Its converting to string type and then asserting